### DISCUSSION TITLE
cannot use path@version syntax in GOPATH mode

### DISCUSSION THREAD

**Discussion by c4ir0** [projectdiscovery/httpx#706] [Q&A] [answered]
command --> go install -v github.com/projectdiscovery/httpx/cmd/httpx@latest

error --> 
can't load package: package github.com/projectdiscovery/httpx/cmd/httpx@latest: cannot use path@version syntax in GOPATH mode

**Comment by lucky0luke** [comment]
try this : 
git clone https://github.com/projectdiscovery/httpx
mv httpx /usr/local/bin

**Reply by forgedhallpass** [reply]
Before moving it, first you would need to build it.

**Reply by lucky0luke** [reply]
yeah sure

**Comment by ehsandeep** [comment, marked as answer]
@c4ir0 most probably you are using outdated version of go, updating go will fix the issue or use above method.
